packages/infra/config/env-config.ts

```diff
@@ -37,6 +37,9 @@ export type EnvConfig = {
   analyticsSecretNames?: {
     POST_HOG_API_KEY: string;
   };
+  placeIndexName: string;
+  placeIndexNameProd?: string;
+  placeIndexProdRegion?: string;
```

Comment:
Let's put all of these under a prop named placeIndex, and always define the name/region accordingly on each env - it doesn't make sense to have 'prod' config on non prod configs